documentation/resource_atlantis_config.md

```diff
@@ -0,0 +1,43 @@
+[back to resource list](https://github.com/sous-chefs/atlantis#resources)
+
+---
+
+# atlantis_config
+
+## Actions
+
+- `:configure` [default]
+- `:remove`
+
+## Properties
+
+| name                        | Type            | Default                     | Description   |
+| --------------------------- | --------------- | --------------------------- | ------------- |
+| atlantis_config_group       | String, Integer | `atlantis`                  |               |
+| atlantis_config_owner       | String, Integer | `atlantis`                  |               |
+| atlantis_config_file        | String          | `atlantis.yml`              |               |
+| atlantis_config_path        | String          | `/opt/atlantis`             |               |
+| atlantis_config_permissions | String, Integer | 0o600                       |               |
```

Comment:
I believe this is an old and not recommended way to show file modes. I think we should instead prefer using strings instead and do this globally: 
```suggestion
| atlantis_config_permissions | String, Integer | '0600'                      |               |
```